prisma-fmt/src/code_actions/mongodb.rs

```diff
@@ -49,3 +49,52 @@ pub(super) fn add_at_map_for_id(
 
     actions.push(CodeActionOrCommand::CodeAction(action))
 }
+
+pub(super) fn add_native_for_auto_id(
+    actions: &mut Vec<CodeActionOrCommand>,
+    params: &lsp_types::CodeActionParams,
+    schema: &str,
+    model: ModelWalker<'_>,
+) {
+    let pk = match model.primary_key() {
+        Some(pk) => pk,
+        None => return,
+    };
+
+    if pk.fields().len() < 1 {
+        return;
+    }
+
+    let field = match pk.fields().next() {
+        Some(field) => field,
+        None => return,
+    };
+
+    let span_diagnostics =
+        match super::diagnostics_for_span(schema, &params.context.diagnostics, model.ast_model().span()) {
+            Some(sd) => sd,
+            None => return,
+        };
+
+    let diagnostics = match super::filter_diagnostics(
+        span_diagnostics,
+        r#"MongoDB `@default(auto())` fields must have `ObjectId` native type."#,
+    ) {
+        Some(value) => value,
+        None => return,
+    };
+
+    let formatted_attribute = super::format_field_attribute(r#"@db.ObjectId"#);
+
+    let edit = super::create_text_edit(schema, formatted_attribute, true, field.ast_field().span(), params);
+
+    let action = CodeAction {
+        title: r#"Add @db.ObjectId"#.to_owned(),
```

Comment:
@Druue it looks like `@db` is hard-coded here? If the database block name is for example `mydb`, the native should start with `@mydb` instead.

Reply:
oh gosh, yeah, I had forgotten that that was the case, I'll change that :)